packages/session-replay-browser/src/session-idb-store.ts

```diff
@@ -57,6 +58,35 @@ export class SessionReplaySessionIDBStore implements AmplitudeSessionReplaySessi
     }
   };
 
+  storeRemoteConfigForSession = async (sessionId: number, remoteConfig: SessionReplayRemoteConfig) => {
```

Comment:
How do we eventually clean these up? I'm thinking this eventually grows into a very large map.

Reply:
We get rid of all session data for sessions older than 3 days here: https://github.com/amplitude/Amplitude-TypeScript/blob/v1.x/packages/session-replay-browser/src/session-idb-store.ts#L111-L111, so this should get removed as a part of that process as well!